### PR TITLE
Power BM hosts down

### PR DESCRIPTION
**Pull request by derekhiggins**
BM hosts need to be powered down so and old
cluster doesn't clash with the new one being
installed

Also change the config example of "NODES_PLATFORM"
to match what is in the README.

### CODE REVIEW

File: 02_configure_host.sh [hunk 1]
```diff
@@ -36,6 +36,23 @@ if [ ! -z "${VM_NODES_FILE}" ]; then
...
 
+# If deploying to a real baremetal environment, we need to make
+# sure all nodes are powered down
+if [[ "${NODES_PLATFORM}" == "baremetal" ]] && [ -n "$NODES_FILE" ] ; then
+    set +x
+    cat $NODES_FILE | jq -r '.nodes[].driver_info | ( .address + " " + .username + " " + .password ) ' |
+    while read ADDRESS USER PASSWORD ; do
+        if [[ $ADDRESS =~ ^ipmi://(.*)$ ]] ; then
+            IPMIIP=${BASH_REMATCH[1]}
+            ipmitool -I lanplus -H $IPMIIP -U $USER -P $PASSWORD power off || echo "WARNING: Skipping power down of $ADDRESS"
```
Comment (andfasano) [inline 02_configure_host.sh:47]: Maybe in case of error it could be useful to print also the error code

Comment (derekhiggins) [reply 02_configure_host.sh]: done

File: 02_configure_host.sh [hunk 2]
```diff
@@ -36,6 +36,23 @@ if [ ! -z "${VM_NODES_FILE}" ]; then
...
 fi
 
+# If deploying to a real baremetal environment, we need to make
+# sure all nodes are powered down
+if [[ "${NODES_PLATFORM}" == "baremetal" ]] && [ -n "$NODES_FILE" ] ; then
+    set +x
+    cat $NODES_FILE | jq -r '.nodes[].driver_info | ( .address + " " + .username + " " + .password ) ' |
+    while read ADDRESS USER PASSWORD ; do
+        if [[ $ADDRESS =~ ^ipmi://(.*)$ ]] ; then
+            IPMIIP=${BASH_REMATCH[1]}
```
Comment (dtantsur) [inline 02_configure_host.sh:46]: This may contain a port, I think

Comment (derekhiggins) [reply 02_configure_host.sh:46]: thanks, fixed

**Pull request merged**